### ISSUE TITLE
crnk-client keep error from server intact

### ISSUE THREAD

**Issue by ryan13mt** [crnk-project/crnk-framework#701]
So user > gateway(crnk-client) > service

Is there a way we can return the exact response/error the gateway receives from the service?

Our issue stems from ClientStubBase.handleError(...) method. This is changing the response and thus we are losing data from our errors.

Basically what i want to achieve is when the service responds with an error, crnk-client returns this exact response to the user.

**Comment by remmeier** [comment]
adding some more fields to the CrnkException would maybe be a possiblity, But this will not work for non-crnk exception. what is your concrete use case?

**Comment by ryan13mt** [comment]
So for example this is the response we receive from the service:

![image](https://user-images.githubusercontent.com/6053618/76194517-a1043000-61e6-11ea-9ba8-d3d309580f1c.png)

The actual response being thrown by the gateway(crnk-client) is this:

![image](https://user-images.githubusercontent.com/6053618/76194633-d90b7300-61e6-11ea-9ab5-398b5d075c2f.png)


Could this issue be stemming from our client side ExceptionMapper implementation?

**Comment by ryan13mt** [comment]
bump @remmeier

**Comment by remmeier** [comment]
yes, currently the gateway would also need crnk-validation to properly serialize/deserialize. But it would very much make sense if it works also otherwise. Meaning CrnkException should hold all the `ErrorData` objects without losing any information. 

This could be a new constructor `public CrnkException(List<ErrorData< message)` and its subtypes?
